backend/copr-backend.spec

```diff
@@ -6,7 +6,7 @@
 %global tests_version 5
 %global tests_tar test-data-copr-backend
 
-%global copr_common_version 0.20.1.dev1
+%global copr_common_version 0.21.1.dev1
```

Comment:
I don't understand changing the versions in this PR so I'll skip this :D

Reply:
Sorry I missed this comment; for the development purposes, we need an NVR >= the_latest_released NVR.  Therefore the bump to 0.21.1 is needed.  I think that 0.21.dev1 woudl be enough, though.